### ISSUE TITLE
Decryption error messages are not actionable and provide little context on what to do next.

### ISSUE THREAD

**Issue by michaelkaye** [vector-im/riot-web#10948]
### Description

I received the error "Unable to decrypt: QuotaExceededError: The current transaction exceeded its quota limitations."

I did the standard refresh the page and the problem persisted.

This error is hard to search for on the searchamatrons, and as a user I don't think I should be expected to read through the fine details of mozilla's developer documentation to find out what to do.

The solution is actually simple: clear out some disk space on your machine, or switch to the application version instead, however you have to investigate the error to provide that.

### Steps to reproduce

- Use riot web, run out of disk space

Describe how what happens differs from what you expected.

Receive the message "Unable to decrypt: QuotaExceededError: The current transaction exceeded its quota limitations." fairly at random, depending on which room you are in.

I would expect an error message which is actionable, such as "Riot does not have sufficient disk space to decode this message" (that triggers me to go and clear some disk space).

Make the error include a link to a error message information section on a website, which could then give guidance on what as a user to do about this error, for instance free up more disk space.

For example, entering in those error codes from personal games consoles: https://support.xbox.com/en-GB/error-code-lookup?code=E12

### Version information

- **Platform**: web 

For the web app:

- **Browser**: Firefox
- **OS**: Linux
- **URL**: riot.im/app

**Comment by turt2live** [comment]
Related: https://github.com/vector-im/riot-web/issues/10891 https://github.com/vector-im/riot-web/issues/6120

**Comment by martindale** [comment]
I too have now bumped into this issue.

**Comment by grinapo** [comment]
Yeah. Extremely useless and unhelpful message, without this issue I wouldn't have figured out what's happenig. (**Disk full**.)
